pkg/app/piped/appconfigreporter/appconfigreporter.go

```diff
@@ -331,8 +332,8 @@ func makeGitPathKey(repoID, cfgFilePath string) string {
 	return fmt.Sprintf("%s:%s", repoID, cfgFilePath)
 }
 
-func (r *Reporter) readApplicationInfo(ctx context.Context, cfgFilePath string) (*model.ApplicationInfo, error) {
-	b, err := fs.ReadFile(r.fileSystem, cfgFilePath)
+func (r *Reporter) readApplicationInfo(appPath, cfgFileAbsPath string) (*model.ApplicationInfo, error) {
```

Comment:
Or `appDirPath, cfgFileRelPath`

Reply:
Sorry, it should be `appDirPath, cfgFilename`

Reply:
I mean we should use the same thing in our code and data in db, they are `appPath` and `cfgFilename`, and `repoDir` is added for dealing with the local file system.